### PR TITLE
Update access-control.rst

### PR DESCRIPTION
**Pull request by bvijayan**
Making text gender neutral

### CODE REVIEW

File: user-guide/access-control.rst [hunk 1]
```diff
@@ -43,9 +43,9 @@ besides validation.
 Client
 ------
 
-A client is a tool or service, that a resource owner wants to use to read or modify his resources. In order to get
-access to the resource owner's resource, the client can ask the resource owner for his consent. If the resource owner
-gives his consent, the client will get a proof that it can forward to the resource server in order to access the
+A client is a tool or service, that a resource owner wants to use to read or modify their resources. In order to get
+access to the resource owner's resource, the client can ask the resource owner for thier consent. If the resource owner
```
Comment (Joneser) [inline user-guide/access-control.rst]: thier -> their

File: user-guide/access-control.rst [hunk 2]
```diff
@@ -24,10 +24,10 @@ Resource Owner
 --------------
 
 The resource owner is typically a human (but doesn't have to be) that owns a resource (data). The resource owner should
-be the only one, who can grant access to his resources.
+be the only one, who can grant access to their resources.
 
-A typical resource owner is a customer, who is the owner of his orders in a shop. Only he should decide, who can access his
-order information. Another example is an employee who owns his salary information.
+A typical resource owner is a customer, who is the owner of thier orders in a shop. Only they should decide, who can access their
```
Comment (Joneser) [inline user-guide/access-control.rst]: thier -> their

Comment (bvijayan) [reply user-guide/access-control.rst]: got it :)

**Comment by hjacobs** [comment]
:+1:

**Comment by Joneser** [comment]
:+1:

**Pull request merged**